.skiff/webapp.jsonnet

```diff
@@ -0,0 +1,235 @@
+    apiVersion: 'v1',
+    kind: 'Namespace',
+    metadata: {
+        name: namespaceName,
+        labels: labels
+    }
+};
+
+local ingress = {
+    apiVersion: 'extensions/v1beta1',
+    kind: 'Ingress',
+    metadata: {
+        name: fullyQualifiedName,
+        namespace: namespaceName,
+        labels: labels,
+        annotations: {
+            'certmanager.k8s.io/cluster-issuer': 'letsencrypt-prod',
+            'kubernetes.io/ingress.class': 'nginx',
+            'nginx.ingress.kubernetes.io/ssl-redirect': 'true',
+            'nginx.ingress.kubernetes.io/enable-cors': 'false'
+        }
+    },
+    spec: {
+        tls: [
+            {
+                secretName: fullyQualifiedName + '-tls',
+                hosts: hosts
+            }
+        ],
+        rules: [
+            {
+                host: host,
+                http: {
+                    paths: [
+                        {
+                            backend: {
+                                serviceName: fullyQualifiedName,
+                                servicePort: config.httpPort
+                            }
+                        }
+                    ]
+                }
+            } for host in hosts
+        ]
+    }
+};
+
+// The `readinessProbe` begins on pod start and determines whether the pod can accept traffic.
+// When an application is not ready traffic will instead go to other ready pods.
+// A live pod can also become unready again if the probe fails `failureThreshold` times.
+local readinessProbe = {
+    failureThreshold: 9,
+    periodSeconds: 30,
+    initialDelaySeconds: 120,  // Use a longer delay if your app loads a large model.
+    httpGet: {
+        path: '/health?check=readiness',
+        port: config.httpPort,
+        scheme: 'HTTP',
+    },
+};
+
+// The `livenessProbe` begins on application start and determines whether to kill the pod.
+// If the `livenessProbe` fails `failureThreshold` times, the pod is killed and restarted by
+// Kubernetes.
+local livenessProbe = {
+    failureThreshold: 6,
+    periodSeconds: 10,
+    initialDelaySeconds: 120,  // Use a longer delay if your app loads a large model.
+    httpGet: {
+        path: '/health?check=liveness',
+        port: config.httpPort,
+        scheme: 'HTTP',
+    },
+};
+
+local deployment = {
+    apiVersion: 'extensions/v1beta1',
+    kind: 'Deployment',
+    metadata: {
+        labels: labels,
+        name: fullyQualifiedName,
+        namespace: namespaceName,
+    },
+    spec: {
+        revisionHistoryLimit: 3,
+        replicas: replicas,
+        template: {
+            metadata: {
+                name: fullyQualifiedName,
+                namespace: namespaceName,
+                labels: labels
+            },
+            spec: {
+                containers: [
+                    {
+                        name: config.appName,
+                        image: image,
+                        args: [ 'server/start.py', '--prod' ],
```

Comment:
The previous [manifest](https://github.com/allenai/lm-explorer/pull/11/files#diff-c13170b96a08fcaf0f67fb4566b08b61L56) didn't provide a command, so I think you'll need to remove this.

Reply:
Thanks--I had thought I'd removed this.